cmd/triggers/create_test.go

```diff
@@ -256,7 +256,7 @@ func TestTriggersCreateCommand(t *testing.T) {
 				appSelectTeardown = setupMockCreateAppSelection(installedProdApp)
 				// TODO: testing chicken and egg: we need the default mocks in place before we can use any of the `clients` methods
```

Comment:
📚 Unrelated to the changes of this PR, we might revisit these comments soon since setting up default mocks is seeming standard **before** customized mocks from what I can tell.

Reply:
🗒️ Good call! I'll add a note to loop back on this after I finish the context work. It almost looks like we can just remove this TODO because it's setting up the default mocks and then adding custom ones. 🤔